report/test/renderer/report-ui-features-test.js

```diff
@@ -702,7 +702,15 @@ describe('ReportUIFeatures', () => {
       Object.values(lhr.categories).forEach(element => {
         element.score = 1;
       });
-      lhr.categories.pwa.score = 0;
+
+      lhr.categories = {
```

Comment:
We should probably just remove the PWA exception here and also remove this test case.